integration/substrate/index.ts

```diff
@@ -25,12 +25,12 @@ export async function createConnection(): Promise<ApiPromise> {
   return ApiPromise.create({ provider: new WsProvider(url) });
 }
 
-export async function deploy(api: ApiPromise, pair: KeyringPair, file: PathLike, ...params: CodecArg[]): Promise<any> {
+export async function deploy(api: ApiPromise, pair: KeyringPair, file: PathLike, value: bigint, ...params: unknown[]): Promise<any> {
```

Comment:
Previously the `deploy` function used the same hard-coded amount for every contract deployed. This amount was set so big that only the first deploy made it, because `Alice` ran out of funds :grin: I assume in previous version the default account had more balance. I thought instead just lowering the amount I would introduce it as an argument. This allows to have different value transfers in every test case (so we could later add tests for things like exceeding `storage_deposit_limit` and possibly other things)